### DISCUSSION TITLE
Outline - Synthetic Data Creation

### DISCUSSION THREAD

**Discussion by kfahn22** [huggingface/computer-vision-course#85] [10. Synthetic Data Creation]
INTRODUCTION (introduction.mdx)  (Assigned to @miniMaddy)

USING A 3D RENDER TO GENERATE  SYNTHETIC DATA (blenderProc.mdx) - first pull request submitted, #63

SYNTHETIC GENERATION USING DCGAN (deep-generative-models.mdx) (Assigned to @kfahn22) 

        DCGAN
        Training
        Evaluation
        Resources
        Notebook (could use some improvement - help welcomed!)

POINT CLOUDS (point-clouds.mdx) (Assigned to @Zekrom-7780) - outline submitted, pull request #70

SYNTHETIC DATA GENERATION USING DIFFUSION MODELS (Assigned to @alperiox)

CHALLENGES WITH USING SYNTHETIC DATA (challenges.mdx) (Assigned to @kfahn22)

      Does the Dataset Adequately Reflect Real World Data?
      Are there Biases in the Synthetic Data?
      Is there Overfitting?
      Does the benefit of using synthetic data justify the computational cost of its generation?

MAJOR SYNTHETIC DATASETS (not an exhaustive list - there are a lot) (Assigned to @WilliamBonvini)

- shapenet
- Pix3D
- FaceSynthetics
- ABO 
- SYNTHIA
- KITTI
- cifake

**NICE ADDITIONS - HELP WANTED:**

PUSHING A DATASET TO THE HUGGING FACE HUB

**Comment by ATaylorAerospace** [comment]
I had a quick question on the major synthetic datasets listed as an example.

Do you think it might be easier to decide which ones to include by breaking them out by Computer Vision Use Case: ie

1.Object Detection and Recognition
YCB-Video

2. Semantic Segmentation:
ApolloScape

3. Autonomous Driving:
GTA5
CARLA

4. Face Recognition:
FFHQ
Craniofacial Datasets

5. Image Super-Resolution:
NTIRE

**Reply by kfahn22** [reply]
Yes that sounds like a good idea!

**Reply by WilliamBonvini** [reply]
I could assign myself the synthetic datasets section. Is anyone else already working on it? It doesn't seem so.

**Reply by kfahn22** [reply]
Great, thanks for working on this!
